brainstormforce/astra
# AST-2232 - Extra margin bottom issue

### Description

Steps To Reproduce: 

- Make sure to have Astra, Astra Pro addon, and Spectra Installed
- Create a page in Gutenberg
- Add Heading block and check the result in frontend

Actual Result:

- A 20px margin-bottom applied in the frontend

Expected Result: 

- There should not be any margin and should look same as backend editor

Attachment and other details: 

A 20px margin is applied after the heading block h2, but there is no control for this.

### Screenshots


### Types of changes


-  Same margin is applied in backend editor & frontend editor now.
- margin-bottom reduced to 10px.




### How has this been tested?

- Make sure to have Astra, Astra Pro addon, and Spectra Installed
- Create a page in Gutenberg
- Add Heading block and check the result in frontend
- Now check in frontend and backend editor margin should be same
### Checklist:
